<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoEnlace</class>
 <widget class="QWidget" name="NuevoEnlace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Camino</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/way-512.png</normaloff>../icons/way-512.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" colspan="3">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Distancia:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>94</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Terminar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Agregar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
